src/e2e-test/features/gcs/sink/GCSSinkError.feature

```diff
@@ -65,3 +65,37 @@ Feature: GCS sink - Verify GCS Sink plugin error scenarios
     Then Select GCS property format "csv"
     Then Click on the Validate button
     Then Verify that the Plugin Property: "format" is displaying an in-line error message: "errorMessageInvalidFormat"
+
+  @GCS_SINK_TEST @BQ_SOURCE_TEST
```

Comment:
Change the tag order, for ease of understanding.

Reply:
Done.